coriolis/wsman.py

```diff
@@ -20,15 +21,16 @@
 
 
 class WSManConnection(object):
-    def __init__(self):
+    def __init__(self, timeout=3600):
         self._protocol = None
+        self._conn_timeout = timeout
 
     EOL = "\r\n"
 
     @utils.retry_on_error()
     def connect(self, url, username, auth=None, password=None,
                 cert_pem=None, cert_key_pem=None):
-        protocol.Protocol.DEFAULT_TIMEOUT = 3600
+        protocol.Protocol.DEFAULT_TIMEOUT = self._conn_timeout
```

Comment:
Considering you found that we can set `protocol.Protocol().transport.timeout` I'd remove this static setting completely and just set that after the `self._protocol` declaration below.